pkg/clients/aws/route53.go

```diff
@@ -72,6 +72,14 @@ func (c *awsClient) GetDNSName() string {
 	return "Route53"
 }
 
+func (c *awsClient) GetFedrampHostedZoneID(fedrampHostedZoneID string) (string, error) {
+	zone, err := c.client.GetHostedZone(&route53.GetHostedZoneInput{Id: &fedrampHostedZoneID})
+	if err != nil {
+		return "", err
+	}
+	return *zone.HostedZone.Id, nil
+}
+
```

Comment:
It looks like this functions purpose is to get the ID of a `hostedZone` from AWS, but we're already providing an argument called `ID`. Are we returning something different than what's provided? If so, can we rename the function or the arguments to make that a bit clearer?